src/Stryker.Core/Stryker.Core/Initialisation/ProjectFileReader.cs

```diff
@@ -35,7 +35,7 @@ public class ProjectFileReader : IProjectFileReader
         public IAnalyzerResult AnalyzeProject(
             string projectFilePath,
             string solutionFilePath,
-            string targetFramework)
+            string targetFramework = null)
```

Comment:
Why this change?

Reply:
This was done to conform with the interface. But in this case I think the interface should be changed rather than the implementation.

Reply:
Hello. That's it. I tried to fix [this code smell](https://sonarcloud.io/code?branch=v1.0&id=stryker-net&selected=stryker-net%3Asrc%2FStryker.Core%2FStryker.Core%2FInitialisation%2FProjectFileReader.cs) (S1006: Method overrides should not change parameter defaults). But it didn't make much sense so I ended up reverting the change.

I'll try changing the interface.

Reply:
Oh, actually it was already done 3 days ago in #1684. Sonarqube analysis was from 16 days ago, that's what got me confused.